jit-compiler/src/codegen.rs

```diff
@@ -211,6 +218,9 @@ impl<'a, T: FieldElement> CodeGenerator<'a, T> {
                     BinaryOperator::ShiftLeft => {
                         format!("(({left}).clone() << u32::try_from(({right}).clone()).unwrap())")
                     }
+                    BinaryOperator::ShiftRight => {
+                        format!("(({left}).clone() >> usize::try_from(({right}).clone()).unwrap())")
```

Comment:
`ShiftLeft` above uses `u32::try_from`, would be good to have consistency

Reply:
true, right somehow didn't work with u32, will check.

Reply:
Changed both to usize, since this is what ibig implements.